.github/workflows/advanced.yml

```diff
@@ -5,6 +5,38 @@ env:
 
 on:
   workflow_dispatch:
+    inputs:
+      openwifi_revision:
+        required: true
+        default: 'main'
+        description: 'revision of the Open Wifi Helm chart'
+      openwifi_gw_revision:
```

Comment:
Think we can remove the sub chart versions on request by Jaspreet.

Reply:
Yep, from what I understand the changes to the deploy script have to be merged first + the blueprint has to be adapted.

Reply:
Don't think so as the Python script will already set some default values, right?

Reply:
yes, depends a bit on whether the ability of testing specific chart versions is required until the changes. If I get it correctly the default values would just overwrite the subchart values every time with master/main regardless of what has been set as the `openwifi_revision` input.